attributes/_install.rb

```diff
@@ -13,8 +13,8 @@
     '/etc/splunk'
   end
 
-default['splunk']['package']['version'] = '7.2.4.2'
-default['splunk']['package']['build'] = 'fb30470262e3'
+default['splunk']['package']['version'] = '7.2.5'
```

Comment:
There appears to be a bug in 7.2.5 with the FIELDALIAS in props.conf, causing search heads to crash[1]. Should we go for the patch version? [2].

[1] https://answers.splunk.com/answers/737075/search-head-crashing-after-upgrade-to-725-due-to-f.html
[2] https://docs.splunk.com/Documentation/Splunk/7.2.5/ReleaseNotes/Fixedissues#Splunk_Enterprise_7.2.5.1

Reply:
Updated to 7.2.5.1 in the latest commit.